paasta_tools/cli/cmds/spark_run.py

```diff
@@ -110,7 +111,12 @@ def add_subparser(subparsers):
         "--image",
         help="Use the provided image to start the Spark driver and executors.",
     )
-
+    list_parser.add_argument(
+        "-a",
+        "--autogen",
+        help="Auto generate pod template with weighted pod affinity used in executors.",
```

Comment:
Description needs to be more descriptive in terms of helping the tool-user to decide a decent value of autogen from the argument itself.
Example: What would happen if autogen is set to 1 vs autogen is set to 100?

Reply:
I might be missing something here. 
From the description here: 
```The weight field in preferredDuringSchedulingIgnoredDuringExecution is in the range 1-100. For each node that meets all of the scheduling requirements (resource request, RequiredDuringScheduling affinity expressions, etc.), the scheduler will compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding MatchExpressions. This score is then combined with the scores of other priority functions for the node. The node(s) with the highest total score are the most preferred.
```, 
it seems like weight field is relevant to make the right choice b/w given set of nodes meeting the criteria of affinity.
AIU, the meeting criteria in our case is the `label` (or app name). And every pod would have same weight (autogen) value.
Would the value of weight matter in that case?

Reply:
you are right, in this sense it would only make sense if the user was running multiple application of the same label with different value then the difference would make sense, I don't mind setting it to a default value in this case